src/Shell/ProcessFactory.php

```diff
@@ -17,7 +17,6 @@ class ProcessFactory
      *
      * @param array $params
      * @return Process|ProcessInterface
-     * @throws \RuntimeException if Process can't be created
```

Comment:
Here can be thrown LogicException.
Doc from `Process::__construct` `* @throws LogicException When proc_open is not installed`